### PR TITLE
 added P1 modal typo toast

### PR DESCRIPTION
**Pull request by swethaogeti**
for preview check : https://swetha-cl-11204d.netlify.app/landingpage.html

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
+.card-container {
+    width: 300px;
```
Comment (Debarshi95) [inline components/card.css:2]: Use rem with width and height

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+.text-card .btn {
+    width: 5rem;
+    margin: 0.5rem;
+}
+
+
+.img-container {
+    width: 160px;
+    height: 200px;
+}
```
Comment (Debarshi95) [inline components/card.css:99]: rems here too.

File: components/utility.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
+*{
+text-decoration: none;
+list-style: none;
+}
```
Comment (Debarshi95) [inline components/utility.css:4]: Using * selector. These style will be applied to all selectors. Are you sure about this?

**Review by kotesh-arya — APPROVED** [review]
Just follow the suggestion by the other folk, rest looks good to me 👍🏻

File: components/utility.css (hunk 3 above)
Comment (swethaogeti) [reply components/utility.css:4]: yes !! actually i wanted to remove all text-decration and list-styling so tried using this !

File: components/card.css [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+   top:15rem;
+}
+
+.text-card .btn {
+    width: 5rem;
+    margin: 0.5rem;
+}
+
+
+.img-container {
```
Comment (theVivek-kumar) [inline components/card.css:96]: use rem here

File: components/card.css [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+    padding: 0.3rem 0.4rem;
+}
+
+.hz-btn-container .btn,
+.btn-ic {
+    margin: 0 0.3rem;
+}
+
+ .card-shadow{
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
```
Comment (theVivek-kumar) [inline components/card.css:159]: use color variables

File: doc-css/doc-component-container.css [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    border-radius: 5px ;
+    border:2px solid var(--grey);
+    flex-wrap: wrap;
+}
+
+.code-display{
+    box-sizing: border-box;
+    display: flex;
+    width: 100%;
+    max-height: 250px;
```
Comment (theVivek-kumar) [inline doc-css/doc-component-container.css:16]: use rem or %

File: doc-css/doc.css [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+body{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    position: relative;
+}
+.landing-page-container {
+    margin: auto;
+    width: 1100px;
```
Comment (theVivek-kumar) [inline doc-css/doc.css:13]: rem

**Review by theVivek-kumar — APPROVED** [review]
looks good to me

**Pull request closed without merging**